src/entity/Session.ts

```diff
@@ -0,0 +1,17 @@
+import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { Action } from './Action';
+
+@Entity()
+export class Session {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@Column()
+	hashed_uuid: string;
+
```

Comment:
missing a start_time or start field (shouldnt be named started_at if we rename ended_at to end_time or end)